Assets/GothicVR/Scripts/Creator/NpcCreator.cs

```diff
@@ -130,8 +130,11 @@ private static void Mdl_SetVisualBody(VmGothicBridge.Mdl_SetVisualBodyData data)
             var npc = lookupCache.npcCache[symbolIndex];
             var mdh = npc.GetComponent<Properties>().mdh;
             var mdm = assetCache.TryGetMdm(data.body);
-            
-            // SingletonBehaviour<MeshCreator>.GetOrCreate().Create(name, mdm, mdh, default, default, npc);
+
+            if (SingletonBehaviour<DebugSettings>.GetOrCreate().EnableNpc)
+                SingletonBehaviour<MeshCreator>.GetOrCreate().Create(name, mdm, mdh, default, default, npc);
+
+            meshCreator.Create(name, mdm, mdh, default, default, npc);
```

Comment:
Please check. It seems you added the same statement twice. Feel free to use this if it makes sense:

```suggestion

            if (SingletonBehaviour<DebugSettings>.GetOrCreate().EnableNpc)
                meshCreator.Create(name, mdm, mdh, default, default, npc);
```

Reply:
my code actually doesnt look like that xD